### ISSUE TITLE
When opening SRT file, sometimes first subtitle times are cleared to zero

### ISSUE THREAD

**Issue by dobratzp** [SubtitleEdit/subtitleedit#8863]
Sometimes when I open an SRT subtitle file, a moment after loading the subtitles, the very first subtitle has its start and end time changed to 00:00:00.000.  When this happens, the SRT file itself is not marked as modified and if you select File => New and then File => Open to open the same SRT file again, it loads fine.

If you don't notice the zeroing out of the first subtitle and then you do additional modifications to the subtitles and then save it, it will persist the 00:00:00.000 to disk.

I haven't figured out the exact sequence of events that causes this to consistently happen.

Subtitle Edit 4.0.8 running on Windows 11 Pro.

**Comment by niksedk** [comment]
Hm, I cannot re-create this... could you make a video + attach the subtitle file?

**Comment by dobratzp** [comment]
I haven't been able to catch the error on video.  Most of the time the error does not occur.  Maybe it's related to timing or caching.  My workflow involves opening an MKV file and selecting a PGS subtitle track to OCR and I've noticed that opening the MKV file may take a few minutes or a few seconds.  Whenever I open an SRT file, it also opens an MKV file in the video player with a similar name.  When I observed the error, I was thinking that it loaded the timestamp correctly from the SRT file and then after the corresponding MKV file loaded in the video player the timestamp of the first subtitle was zeroed out.
